src/handlers/metadataDelta/utils.ts

```diff
@@ -0,0 +1,42 @@
+import { verbose } from '~utils';
+import {
+  AddVerifiedMembersOperation,
+  Entity,
+  OperationType,
+  MetadataDeltaOperation,
+} from './types';
+
+export const isAddVerifiedMembersOperation = (
+  operation: MetadataDeltaOperation,
+): operation is AddVerifiedMembersOperation => {
+  return (
+    operation.entity === Entity.VERIFIED_MEMBERS &&
+    operation.type === OperationType.ADD
+  );
+};
+
+export const isMetadataDeltaOperation = (
+  operation: any,
+): operation is MetadataDeltaOperation => {
+  return (
+    typeof operation === 'object' &&
+    operation !== null &&
+    operation.type !== undefined &&
+    operation.entity !== undefined &&
+    operation.colonyAddress !== undefined &&
+    typeof operation.colonyAddress === 'string' &&
+    operation.members !== undefined &&
+    Array.isArray(operation.members)
```

Comment:
It looks like there's a conflict between trying to make this generic (operating on an arbitrary Metadata Delta Operation) and being specific to verified members (sounds possible that other operations won't have members field?)

Would it not be easier if your general Metadata Delta Operation would only be concerned with type (Add Members, Remove Members) and payload specific for a type? I also think you don't need colony address here, it's tied to the event itself.

The job of this handler could then be reduced to:
a) check for type of operation, ignore it if it's unknown.
b) forward the operation to an operation-specific handler/util, which could do validation on a per-operation basis.

All operation types could be nicely extracted, similar to how we handle different types of actions in `motionCreated`.

A sample metadata could look like so:
```json
{
  "type": "ADD_VERIFIED_MEMBERS",
  "payload": ["0x...", "0x..."]
}
```